### ISSUE TITLE
Pretrained models

### ISSUE THREAD

**Issue by mesnico** [chunxiaoliu6/BFAN#1]
Thank you for the nice repo. I am trying to evaluate your model on a different metric. 
Can you please share the pre-trained models? Thank you!
